File: Cypress/UI-Testing.CypressTests/cypress/integration/tag/tagPageTests.js
describe('When visiting tag page', () => {
    it('Then verify Umbraco tag page is accessible', () => {
        cy.visit('https://adolfi.dev/tags/umbraco');  
    });

    it('Then verify Unit Testing tag page is accessible', () => {
        cy.visit('https://adolfi.dev/tags/unit%20testing');  
    });

    it('Then verify .NET tag page is accessible', () => {
        cy.visit('https://adolfi.dev/tags/-net');  
    });
})

File: Playwright/UI-Testing.PlaywrightTests/Tests/Tag/TagPageTests.cs
using Microsoft.Playwright.NUnit;
using NUnit.Framework;
using System.Threading.Tasks;

namespace UI_Testing.PlaywrightTests.Tests.Tag
{
    public class TagPageTests : PageTest
    {
        [Test]
        public async Task Given_NavigateToUmbracoTags_Then_AssertLocation()
        {
            // When
            await Page.GotoAsync("https://adolfi.dev/tags/umbraco");

            // Then
            Assert.AreEqual("https://adolfi.dev/tags/umbraco", Page.Url);
        }

        [Test]
        public async Task Given_NavigateToUnitTestingTags_Then_AssertLocation()
        {
            // When
            await Page.GotoAsync("https://adolfi.dev/tags/unit%20testing");

            // Then
            Assert.AreEqual("https://adolfi.dev/tags/unit%20testing", Page.Url);
        }

        [Test]
        public async Task Given_NavigateToDotNetTags_Then_AssertLocation()
        {
            // When
            await Page.GotoAsync("https://adolfi.dev/tags/-net");

            // Then
            Assert.AreEqual("https://adolfi.dev/tags/-net", Page.Url);
        }
    }
}
